<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_LessonAdd</class>
 <widget class="QWidget" name="Widget_LessonAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称（必填）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Name">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>30</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_location">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>上课地点</string>
   </property>
   <widget class="QRadioButton" name="radio_xitucheng">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>西土城校区</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_shahe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>沙河校区</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上课星期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上课节数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上课班级（必修填）</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_week">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_index">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_class">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_c">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加必修课程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_s">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加选修课程</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qun">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>80</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>课程群号（选填）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_extra">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
